rollup.config.js

```diff
@@ -1,5 +1,5 @@
 const path = require('path');
-const resolve = require('@rollup/plugin-node-resolve');
+import resolve from '@rollup/plugin-node-resolve';
```

Comment:
As I know `import` and `require()` can't be used in a single module. I would be surprised if this works ...

Reply:
They do work together, at least on the latest Node which is 14.3.0
The reason why I changed `require()` to `import` is that I got this error after upgrading the plugin:

```
$ rollup -c
[!] TypeError: resolve is not a function
TypeError: resolve is not a function
    at Object.<anonymous> (rollup.config.js:30:5)
    at Module._compile (internal/modules/cjs/loader.js:1200:30)
    at Object.require.extensions.<computed> [as .js] (node_modules\rollup\dist\shared\loadConfigFile.js:516:20)
    at Module.load (internal/modules/cjs/loader.js:1049:32)
    at Function.Module._load (internal/modules/cjs/loader.js:937:14)
    at Module.require (internal/modules/cjs/loader.js:1089:19)
    at require (internal/modules/cjs/helpers.js:73:18)
    at loadConfigFromBundledFile (node_modules\rollup\dist\shared\loadConfigFile.js:524:42)
    at getDefaultFromTranspiledConfigFile (node_modules\rollup\dist\shared\loadConfigFile.js:509:12)
    at loadConfigFile (node_modules\rollup\dist\shared\loadConfigFile.js:487:15)

error Command failed with exit code 1.
```

rollup is 2.10.9